master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -408,6 +421,19 @@ private[celeborn] class Master(
         context,
         handleReportNodeUnavailable(context, failedWorkers, requestId))
 
+    case pb: PbWorkerExclude =>
+      val workersToAdd = new util.ArrayList[WorkerInfo](pb.getWorkersToAddList
+        .asScala.map(PbSerDeUtils.fromPbWorkerInfo).toList.asJava)
+      val workersToRemove = new util.ArrayList[WorkerInfo](pb.getWorkersToRemoveList
+        .asScala.map(PbSerDeUtils.fromPbWorkerInfo).toList.asJava)
+      executeWithLeaderChecker(
+        null,
+        handleWorkerExclude(
+          null,
```

Comment:
Context should be added here to make sure that excluded responses can be returned.